src/asset/asset.js

```diff
@@ -70,7 +70,7 @@ class Asset extends EventHandler {
      * field or null if hash was set (e.g from using {@link AssetRegistry#loadFromUrl}).
      * @param {ArrayBuffer} [file.contents] - Optional file contents. This is faster than wrapping
      * the data in a (base64 encoded) blob. Currently only used by container assets.
-     * @param {object} [data] - JSON object with additional data about the asset. (e.g. for texture
+     * @param {object|string} [data] - JSON object or string with additional data about the asset. (e.g. for texture
```

Comment:
Note that all JSDoc comments fit in 100 columns now (I formatted the entire codebase that way). Does this line now go over that?